### ISSUE TITLE
Layout empty not found.

### ISSUE THREAD

**Issue by diennttlu** [abpframework/abp#17690]
In route.provider.ts
```
{
        path: '/appsetting/generate',
        name: '::AppSetting:Generator',
        parentName: '::Menu:AppSetting',
        order: 2,
        layout: eLayoutType.empty
 }
```
My component: 
![image](https://github.com/abpframework/abp/assets/36124566/2f3f248d-5715-48fc-aa7c-7a81d2a654ec)
Why not show content?
![image](https://github.com/abpframework/abp/assets/36124566/e6eb1ed8-d735-407c-ae3d-c4d483bc5600)

ABP Framework 6.0.0.0
UI Angular  @abp/ng.theme.lepton-x
DB EF Core

**Comment by diennttlu** [comment]
please, help me!

**Comment by mkmita** [comment]
Is there any work around i think i have same issue here with lepton-x commercial?

**Comment by Sinan997** [comment]
we are working on it, we will inform you

**Comment by diennttlu** [comment]
Can you show us your sample repository?

**Comment by Sinan997** [comment]
Hi @diennttlu , @mkmita 
You can use this as a workaround, we will fix this in 7.4 patch

**imports**
```ts
import { APP_INITIALIZER } from '@angular/core';
import { ReplaceableComponentsService } from '@abp/ng.core';
import { eThemeLeptonXComponents } from '@volosoft/abp.ng.theme.lepton-x';

```


app.module.ts 
```ts
providers: [
  ...
  {
    provide:APP_INITIALIZER,
    useFactory: initEmptyLayout,
    multi: true
  }
]
```

```ts
export function initEmptyLayout() {
  const replaceableComponents = inject(ReplaceableComponentsService);
  return function(){
    replaceableComponents.add({
      key: eThemeLeptonXComponents.EmptyLayout,
      component: EmptyLayoutComponent,
    });
  }
}
```


**Create this emptyLayoutComponent**
```ts
import { Component } from '@angular/core';
import { CoreModule, eLayoutType } from '@abp/ng.core';
import { ThemeSharedModule } from '@abp/ng.theme.shared';

@Component({
  selector: 'abp-layout-empty',
  standalone: true,
  imports: [CoreModule, ThemeSharedModule],
  template: `
    <router-outlet></router-outlet>
    <abp-confirmation></abp-confirmation>
  `,
})
export class EmptyLayoutComponent {
  static type = eLayoutType.empty;
}

```

**Comment by diennttlu** [comment]
Oh. good
thanks so much!
